raiden-ts/src/helpers.ts

```diff
@@ -525,11 +525,11 @@ export async function getState(
   ].join('_');
 
   let db;
-  let { state: dump } = storage;
-  const { adapter, prefix } = storage;
+  const { state: stateDump, ...opts } = storage;
```

Comment:
Weren't these changes already committed? Maybe a rebase is needed.

Reply:
This branch branched from master way before your change. I just cherry-picked it in.